### PR TITLE
User phase

### LINKED ISSUE #34: Implement pool-based AL
Implement pool-based AL, using one of the AL-algorithm, this implementation should work in an general way for different AL-algorithms.

Requires one implemented AL-algorithm: #33  or #32  or #31  or #29 etc

See #7

### LINKED ISSUE #62: User Phase
Should perform Active Learning with user for the best sets as determined by #61. The results should be fetched from #65.

### PR DESCRIPTION
**Pull request by sporreking**
Closes #34 
Closes #62

### CODE REVIEW

**Review by caspiansusskind — CHANGES_REQUESTED** [review]
Looks good! I left some comments about minor things that we should probably fix.

File: src/init_phase.py [hunk 1]
```diff
@@ -8,10 +8,8 @@
 import numpy as np
 
 
-BATCH_SIZE = 100
-
-
-def init_phase(sliding_window_length, model_eval=False):
+def init_phase(data_dir: Path, sliding_window_length: int,
+               batch_size: int = 100, model_eval: bool = False):
```
[the remark this answers is not in the document]
Comment (sporreking) [reply src/init_phase.py:12]: Good catch! I had missed this in another place as well.

File: src/init_phase.py [hunk 2]
```diff
@@ -137,18 +135,18 @@ def load_train_test_datasets(id, train, test, feat_sel):
         f"ds{id}_train",
         Path(
             f"res/data/features_librosa_{train}_{feat_sel}.npy"),
```
[the remark this answers is not in the document]
Comment (sporreking) [reply src/init_phase.py]: Indeed!

**Review by caspiansusskind — APPROVED** [review]
Looks great!

**Pull request merged**